### PR TITLE
Added an entry for ClamWin antivirus version 0.99.4. I tested this on…

### PR DESCRIPTION
**Pull request by qbancoffee**
… reactos 0.4.13 and it worked well.

Soon after it intalls it begins to download virus definitions and it looks like it's hanging but after waiting
a couple of minutes the virus definitions download and the downloader window becomes responsive.

A reboot is needed for ClamWin to show in the menu

### CODE REVIEW

**Comment by HBelusca** [comment]
But does it actually remove / fix infected files?

**Comment by qbancoffee** [comment]
> But does it actually remove / fix infected files?

Good question. 
Once configured, yes, I created a test file following these instructions.
https://www.makeuseof.com/tag/how-to-safely-test-your-antivirus-software/

**Comment by HBelusca** [comment]
Well, the EICAR test doesn't completely test everything.

**Comment by qbancoffee** [comment]
Ill have to do some more tests then. Aside from actually infection , are
you aware of a more robust test?

**Comment by HBelusca** [comment]
I don't have. I was just thinking about putting one or two actual viruses in ReactOS and let them infect some files, and then check how clamwin behaves during disinfection (whether it does anything, or makes ReactOS crash).

**Comment by qbancoffee** [comment]
I was able do download actual infected files from a malware database website and WinClam could detect them. I'll have to do some more testing but actually trying to infect reactos is turning out to be a little harder than I though.

Here is screenshot.
virus_detected_1

**Comment by HBelusca** [comment]
Looks fine, thanks!

File: clamwin.txt [hunk 1]
```diff
@@ -0,0 +1,16 @@
+[Section]
+Name = ClamWin Free Antivirus
+Version = 0.99.4
+License = GPL
+LicenseInfo = 1
+Description =  ClamWin is a Free Antivirus program for Microsoft Windows 10/8/7/Vista/XP/Me/2000/98 and Windows Server 2012, 2008 and 2003.
```
Comment (HBelusca) [inline clamwin.txt:6]:
```suggestion
Description = ClamWin is a Free Antivirus program for Microsoft Windows 10/8/7/Vista/XP/Me/2000/98 and Windows Server 2012, 2008 and 2003.
```

File: clamwin.txt [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+Category = 12
+URLSite = http://www.clamwin.com/
+URLDownload = https://sourceforge.net/projects/clamwin/files/clamwin/0.99.4/clamwin-0.99.4-setup.exe
+SHA1 = c9e89a9ec6351545b6dfb3f0c9a0ff41660ce09c
+SizeBytes = 172661090
+Languages=0409|040A
```
Comment (HBelusca) [inline clamwin.txt:12]:
```suggestion
Languages = 0409|040A
```

**Review by HBelusca — COMMENTED** [review]
Minor formatting fixes.

**Pull request merged**